### ISSUE TITLE
soundpipe.h file not found

### ISSUE THREAD

**Issue by jbmaxwell** [AudioKit/AudioKitSynthOne#158]
I'm trying to move our "skinless" version of Synth One into a new SwiftUI project, but I keep hitting the above error (soundpipe.h file not found). I can see that soundpipe.h is part of AudioKit, so I'm not sure why it isn't being recognized. Any advice? I've been struggling with this for a few hours...

[EDIT: The error appears in oscmorph2d.c, by the way.]

**Comment by aure** [comment]
You're doing something wrong is my snarky answer, but really how can we help with so little to go on. You're asking a question here, but this is a forum for identifying issues in our code. Maybe try StackOverflow? I'm sure the answer has to do with how you're including files or something, but without the Xcode project, I can't fathom to guess which magic config thing needs be changed. Xcode is quite a complicated beast after all. Anyway, I am glad that you're trying to use the engine for other projects, we just can't help without more to go on, I hope you understand and are not offended by my snark! :)

**Comment by jbmaxwell** [comment]
Unfortunately I've had very little luck getting AudioKit questions answered on SO in the past—particularly more specialized ones like this. And since this isn't exactly an AudioKit question, I thought that with a (much) smaller group of people, likely working much closer to the code, I'd have better luck here. Also, yes, Xcode is insanely complicated, and I have limited experience working with projects that integrate Swift, C++, and C, and it seemed like the kind of problem someone working closely on the project might have a strong intuition about solving. I've found it very helpful in the past to find these kinds of issues (or rather, their resolutions) on GitHub, as answers to unusual problems often come from devs directly.
At any rate, I did get it to build late last night. The problem (as you suggested) was indeed to do with oscmorph2d.c being assigned a target by Xcode when it was added to the project. It seems that file is used during the compilation of another file (or files), but shouldn't be assigned to a target itself. Anyone having a similar problem, in future, might save themselves a few hours with that info.
